src/Esfa.Vacancy.Manage.Api/Controllers/CreateApprenticeshipController.cs

```diff
@@ -40,6 +40,11 @@ public CreateApprenticeshipController(CreateApprenticeshipOrchestrator orchestra
         /// | 31012       | Expected start date is required                                                  |
         /// | 31013       | Expected start date must be after application closing date                       |
         /// | 31014       | The request must be valid json/xml and contain required values                   |
+        /// | 31015       | Working week can't be empty                                                      |
```

Comment:
Working week is required*****

Reply:
I based it on the other fields. Admittedly the 2 dates fields don't follow this pattern but I suggest that they all are the same for consistency.